mozilla_general/README.md

```diff
@@ -36,41 +30,44 @@ At times there will be English content included in Mozilla products or web proje
 
 ### Abbreviations and acronyms
 
-Abbreviations and acronyms, should come from either a standard reference (like a dictionary of abbreviations) or should follow your language's rules for creating abbreviations and acronyms. Once used, abbreviations and acronyms must remain consistent every place that it is used in the translation.
-
-> Link to list of common abbreviations in Mozilla products
-
-> Link to list of common Acronyms in Mozilla products
+Abbreviations and acronyms, should come from either a standard reference (like a dictionary of abbreviations) or should follow your language’s rules for creating abbreviations and acronyms. Once used, abbreviations and acronyms must remain consistent every place that it is used in the translation.
 
 ### Branding
 
-Adherence to [Mozilla and third-party branding and style guides](https://www.mozilla.org/en-US/styleguide/identity/firefox/branding/) should be respected throughout a localization project. For example, some brand names should never be translated, such as `Firefox`. For other brands that do not have any branding guidelines, your localization community must define whether to translate them. Be extra careful to check on branding rules before deciding to translate a name or not (whether for Mozilla or for a third-party) and to list them here in your community's l10n style guide.
+Adherence to [Mozilla and third-party branding and style guides](https://www.mozilla.org/en-US/styleguide/identity/firefox/branding/) should be respected throughout a localization project. For example, some brand names should never be translated, such as `Firefox`. For other brands that do not have any branding guidelines, your localization community must define whether to translate them. Be extra careful to check on branding rules before deciding to translate a name or not (whether for Mozilla or for a third-party) and to list them here in your community’s l10n style guide.
 
 ## Text that should not be translated
 
 ### Shortcuts and accesskeys
 
-In Firefox and other software it's possible to use keyboard shortcuts to invoke a specific command. For example, to open a file in Firefox you can press the combination of keys CTRL+O (Cmd+O on Mac). The accelerator key depends on the operative system, but the letter itself is normally localizable. This is what is called a shortcut, or commandkey. For example, the Open File… menu item is stored as
+In Firefox and other software it’s possible to use keyboard shortcuts to invoke a specific command. For example, to open a file in Firefox you can press the combination of keys CTRL+O (Cmd+O on Mac). The accelerator key depends on the operative system, but the letter itself is normally localizable. This is what is called a shortcut, or commandkey. For example, the `Open File…`` menu item is stored as
```

Comment:
Extra "`" after Open File…
Also first time I read operative system, but I will assume it’s my non-native speaker brain failing